framework/src/play/db/Configuration.java

```diff
@@ -98,7 +98,7 @@ public Object put(String key, String value) {
   }
 
   String generateKey(String key) {
-    Pattern pattern = Pattern.compile(regexDbRelatedSettings + "(\\.?[\\da-zA-Z\\.-_]*)$");
+    Pattern pattern = Pattern.compile(regexDbRelatedSettings + "(\\.?[\\da-zA-Z.-_]*)$");
```

Comment:
What is the motivation behind this change?

Reply:
that the `\\` is useless there.